src/types/ProcessedMessage.ts

```diff
@@ -0,0 +1,21 @@
+import type { Address } from "abitype";
+import type { Hex } from "viem";
```

Comment:
use our type

Reply:
this is important because we will remove viem

Reply:
updated look please